.gitignore

```diff
@@ -27,3 +27,6 @@ npm-debug.log*
 *.njsproj
 *.sln
 *.sw?
+
+/tmp/
```

Comment:
What do you think about using `/nginx/` instead of `/tmp/`? We haven't needed a `tmp` directory for other reasons, and if this directory is only going to hold `nginx` files, maybe `/nginx/` would be more immediately understandable as a name. 🤔

Reply:
How about `.nginx` - i think the dot prefix would make it clearer it's a temporary directory